packages/core/package.json

```diff
@@ -65,10 +65,8 @@
     "inngest": "3.22.3",
     "radash": "^12.1.0",
     "sift": "^17.1.3",
-    "xstate": "^5.19.0"
-  },
-  "peerDependencies": {
-    "zod": "3.23.7"
+    "xstate": "^5.19.0",
```

Comment:
Not sure why we are adding xstate as peerDependencies. I don’t think we should prompt the users of Mastra to install xstate.

Reply:
i see you've modified this on your PR so I am gonna be reverting